city/models.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import Column, Integer, String
+from database import Base
+
+
+class City(Base):
+    __tablename__ = "city"
+    id: int = Column(Integer, primary_key=True, index=True)
+    name = Column(String(255), nullable=False)
+    additional_info = Column(String(500), nullable=False)
```

Comment:
The `additional_info` column is set to `nullable=False`. Ensure that this field is required for every city entry. If not, consider setting `nullable=True` to allow entries without additional information.